### DISCUSSION TITLE
LORA promised small(4MB) training file, but currently I get the old 4.3GB ckpt, why?

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [d8ahazard/sd_dreambooth_extension#537] [General]
LORA promised small(4MB) training file, but currently I get the old 4.3GB ckpt, why?
Are small files supported at all by webui?
Where are they stored?

**Comment by Thomas-MMJ** [comment]
Currently the script generates ckpts but you can disable it in the code (there is also a patch to disable it - see the pull request tracker).

You can't load LORAs in web-ui currently, so to use them with automatic1111 web-ui you will have to create a checkpoint eventually.

**Comment by a-l-e-x-d-s-9** [comment]
I think that LORA's small model format could be a good addition to webui.
